jobrunner/manage_jobs.py

```diff
@@ -269,6 +269,7 @@ def finalise_job(job):
     container_metadata = get_container_metadata(job)
     outputs, unmatched_patterns = find_matching_outputs(job)
     job.outputs = outputs
+    job.image_id = container_metadata["Id"]
```

Comment:
Ah, confusingly that's the _container_ ID. We want the ID of the image used to run it:
https://github.com/opensafely-core/job-runner/blob/5e421c4164369a14d46eeee333e14895f2988363/jobrunner/manage_jobs.py#L426

Reply:
Doh! Fixed.